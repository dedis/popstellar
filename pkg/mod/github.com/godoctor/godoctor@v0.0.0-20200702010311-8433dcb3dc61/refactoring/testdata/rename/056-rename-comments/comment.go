package tmp2
var q int //q, yes q. The name q appears four times in this quick comment for q // <<<<< rename,2,5,2,5,xx,pass
             
