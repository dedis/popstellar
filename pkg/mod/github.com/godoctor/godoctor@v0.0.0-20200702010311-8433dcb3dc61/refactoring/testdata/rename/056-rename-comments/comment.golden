package tmp2
var xx int //xx, yes xx. The name xx appears four times in this quick comment for xx // <<<<< rename,2,5,2,5,xx,pass
             
